hoot-core-test/src/test/cpp/hoot/core/io/OsmPbfReaderTest.cpp

```diff
@@ -439,7 +439,7 @@ class OsmPbfReaderTest : public HootTestFixture
     OsmPbfReader reader(false);
     const int chunkSize = 40;
     reader.setMaxElementsPerMap(chunkSize);
-    reader.open("test-files/ToyTestCombined.pbf");
+    reader.open("test-files/ToyTestCombined.osm.pbf");
```

Comment:
I renamed a few files because technically they should be `.osm.pbf` from the [wiki](https://wiki.openstreetmap.org/wiki/PBF_Format).